transport/types/types.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+import (
+	"github.com/filecoin-project/boost/transport/types/transferstatus"
+	"github.com/google/uuid"
+)
+
+type TransportDealInfo struct {
```

Comment:
where will this be set up? how is output file name chosen? will the file name need to be sanitized?